src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js

```diff
@@ -267,6 +269,33 @@ export default class SpawnPatternModel extends BaseModel {
     }
 
     /**
+     * Destroy the current instance properties
+     *
+     * Useful when changing airports
+     *
+     * @for SpawnPatternModel
+     * @method destroy
+     */
+    destroy() {
```

Comment:
What's the difference between when you're sometimes using `reset` and `destroy` with destructive methods such as these?